Project 1 (9.12)/theme switcher/styles.css

```diff
@@ -8,4 +8,32 @@
     cursor: pointer;
     font-size: 1rem;
     padding: 0.75rem 1.25rem;
+  }
+
+
+
+  .lightActive{
+    background-color: white;
+  }
+  
+  .darkActive{
+   background-color: black;
+  
+  }
+
+  /* .btnDark{
+    background-color: white;
+    color: black;
+  } */
+  .btnDark{
+   border-color: white;
```

Comment:
nie jest to zbyt semantyczne (button dark, a kolor white :)), poza tym przycisk jest niewidoczny po odpaleniu w przeglądarce :)